.github/workflows/examples_publish.yaml

```diff
@@ -68,4 +71,34 @@ jobs:
           file: ./${{ matrix.dockerfile }}/Dockerfile
           platforms: ${{ matrix.platforms }}
           push: true
-          tags: ${{ steps.prep.outputs.tags }}
\ No newline at end of file
+          tags: ${{ steps.prep.outputs.tags }}
+      - name: Export cos version
+        run: |
+             source .github/helpers.sh
+             echo "COS_VERSION=$(cos_version)" >> $GITHUB_ENV
+      - name: Build Image
+        id: build_image
+        run: |
+          mkdir build
+          sudo ./images/arm-img-builder.sh --docker-image ${{ steps.prep.outputs.current }} build/cOS-example-${{ matrix.tag }}$COS_VERSION.img
+      - name: Upload Image
+        run: |
+          sudo tar cvf build.tar build
+          sudo luet util pack quay.io/costoolkit/examples:${{ matrix.tag }}$COS_VERSION.img build.tar image.tar
```

Comment:
This creates a docker image for the raw `img` file. I suspect those files are too big to be released over Github, so we might end up just distributing images that can be pulled locally